Resources/public/js/views/services/ez-locationviewviewservice.js

```diff
@@ -43,6 +44,136 @@ YUI.add('ez-locationviewviewservice', function (Y) {
         },
 
         /**
+         * sendToTrashAction event handler, sends location available in request to trash
+         * and makes the application navigate to parent location
+         *
+         * @method _sendContentToTrash
+         * @protected
+         * @param {Object} e event facade of the sendToTrashAction event
+         */
+        _sendContentToTrash: function (e) {
+            var app = this.get('app'),
+                that = this,
+                contentService = this.get('capi').getContentService(),
+                location = this.get('location'),
+                locationId = location.get('id'),
+                contentName = e.content.get('name');
+
+            e.preventDefault();
+            this._set('trackOutsideEvents', false);
```

Comment:
there's `trackOutsideEvents` attribute on LocationViewViewService, I guess it's a copy paste from UniversalDiscoveryConfirmedListView, so this should be removed (and below as well).